Documentation/UsingSetting/Index.rst

```diff
@@ -436,6 +436,7 @@ The PHP API to retrieve page and user TSconfig in a backend module is (nowadays)
 
 .. code-block:: php
 
+    // use TYPO3\CMS\Backend\Utility\BackendUtility;
```

Comment:
Yes, we have already added some use statements in other code examples.

To make this stand out more, I would not make it a comment and put it in a separate block. (Currently, it looks like part of the comment).